docs/platforms/android/techniques_operable_android.md

```diff
@@ -1,15 +1,10 @@
- [🔼 Accessibility principles and examples](../../principles/accessibility_principles_and_examples.md "Accessibility principles and examples") | [⬅️ Operable principle](../../principles/operable_principle.md "Operable principle")
-# Operable guidelines for Android
+# [2. Operable principle](../../principles/operable_principle.md#2-operable-principle)
 
-User interface components and navigation must be operable.
+## [2.1. Keyboard Accessible](../../principles/operable_principle.md#21-keyboard-accessible)
 
-## Keyboard (WCAG 2.1.1 - Level A)
```

Comment:
I would recommend to avoid removing `WCAG <version>` because this is just one of the standards we may need; e.g. WAD is another one which we will need to handle.

Also, for the reference, we will need to know which version of standard are we referring.

Reply:
I renamed `Keyboard (WCAG 2.1.1 - Level A)` to `2.1.1 Keyboard (Level A)`. For better readability I moved the number to the start.

Reply:
I do undestand your point, but that can make things confusing, because those numbers are not our chapters, it is only a referal point to the WCAG standard. Also, we won't cover all "chapters" from WCAG and it could look like we skipped someting.

This is why I would put that as it was earlier - we do not use it as a chapter definition, rather as a WCAG reference - Keyboard (WCAG 2.1.1 - Level A). In future there can be something else (e.g. WAD regulation or something) that will have different identification.

Reply:
Before I go changing the titles back. Do we even need to have the WCAG reference in the title?
Can the title be just `Keyboard`?
We do have the following text in the description:
> This guideline with provided techniques covers the 1.1.1 Non-text Content - Level A of the WCAG standard.

Reply:
I do understand your point, but for the easier navigation (due to usage of headings), people will be easier to navigate just based on title. If we remove it, people will need to read the description which says to which WCAG chapter this one refers.

Reply:
Ok, I will leave the WCAG reference in the title, so `Keyboard (WCAG 2.1.1 - Level A)`